packages/terra-progress-bar/src/ProgressBar.jsx

```diff
@@ -70,19 +71,23 @@ const ProgressBar = ({
   );
 
   const normalizedValue = (value / max) * 100;
+  const valText = valueText || `${normalizedValue}%`;
```

Comment:
percentage symbol `%` needs to be evaluated with translation team to ensure it conveys the same meaning on different locales